cmd/ddev/cmd/dev_sequelpro.go

```diff
@@ -6,69 +6,107 @@ import (
+// SequelproLoc is where we expect to find the sequel pro.app
+// It's global so it can be mocked in testing.
+var SequelproLoc = "/Applications/sequel pro.app"
+
+// localDevSequelproCmd represents the sequelpro command
+var localDevSequelproCmd = &cobra.Command{
 	Use:   "sequelpro",
 	Short: "Easily connect local site to sequelpro",
-	Long:  `A helper command for easily using sequelpro with a drud app that has been initialized locally.`,
+	Long:  `A helper command for easily using sequelpro (OSX database browser) with a ddev app that has been initialized locally.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		app, err := getActiveApp()
+		out, err := handleSequelProCommand(SequelproLoc, args)
 		if err != nil {
-			log.Fatalf("Could not find an active ddev configuration, have you run 'ddev config'?: %v", err)
+			log.Fatalf("Could not run sequelpro command: %s", err)
 		}
+		util.Success(out)
+	},
+}
 
-		nameContainer := fmt.Sprintf("%s-db", app.ContainerName())
+// handleSequelProCommand() is the "real" handler for the real command
+func handleSequelProCommand(appLocation string, args []string) (string, error) {
+	app, err := getActiveApp()
+	if err != nil {
+		return "", err
+	}
 
-		if !dockerutil.IsRunning(nameContainer) {
-			util.Failed("App not running locally. Try `ddev start`.")
-		}
+	if len(args) != 0 {
+		return "", fmt.Errorf("invalid arguments to sequelpro command: %v", args)
+	}
+	nameContainer := fmt.Sprintf("%s-db", app.ContainerName())
 
-		mysqlContainer, err := dockerutil.GetContainer(nameContainer)
-		if err != nil {
-			log.Fatal(err)
-		}
+	if !dockerutil.IsRunning(nameContainer) {
+		return "", errors.New("app not running locally. Try `ddev start`")
+	}
 
-		dbPort, err := dockerutil.GetDockerPublicPort(mysqlContainer, int64(3306))
-		if err != nil {
-			log.Fatal(err)
-		}
+	mysqlContainer, err := dockerutil.GetContainer(nameContainer)
+	if err != nil {
+		return "", err
+	}
 
-		tmpFilePath := path.Join(app.AppRoot(), "sequelpro.spf")
-		tmpFile, err := os.Create(tmpFilePath)
-		if err != nil {
-			log.Fatalln(err)
-		}
-		defer util.CheckClose(tmpFile)
-
-		_, err = tmpFile.WriteString(fmt.Sprintf(
-			platform.SequelproTemplate,
-			"data",                  //dbname
-			"127.0.0.1",             //host
-			mysqlContainer.Names[0], //container name
-			"root",                  // dbpass
-			strconv.FormatInt(dbPort, 10), // port
-			"root", //dbuser
-		))
-		util.CheckErr(err)
-
-		err = exec.Command("open", tmpFilePath).Run()
-		if err != nil {
-			log.Fatal(err)
-		}
+	dbPort := appports.GetPort("db")
+
+	tmpFilePath := path.Join(app.AppRoot(), ".ddev/sequelpro.spf")
+	tmpFile, err := os.Create(tmpFilePath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	defer util.CheckClose(tmpFile)
+
+	_, err = tmpFile.WriteString(fmt.Sprintf(
+		platform.SequelproTemplate,
+		"data",                  //dbname
+		app.HostName(),          //host
+		mysqlContainer.Names[0], //container name
```

Comment:
Or better yet, perhaps this value should just be the `app.HostName()` value? probably?